src/puppetlabs/trapperkeeper/services/webserver/jetty10_service.clj

```diff
@@ -49,7 +49,19 @@
                           ;; Here for backward compatibility with existing projects
                           (tk-config/get-in-config config-service [:jetty])
                           {})
+               _ (log/debug (i18n/trs "Jetty server(s) starting with config: {0}" config))
                started-context (core/start! context config)]
+           ;; Log started server(s) configuration(s) for debugging purposes
+           (doseq [started-server-context (:jetty10-servers started-context)]
+             ;; Server context data is in the form {:servername {:handlers <handlers>, ... }}
+             (let [server-name (name (first started-server-context))
+                   started-server (:server (second started-server-context))
+                   debug-dump (java.io.StringWriter.)]
```

Comment:
I think we would want to use a "with-open" on the writer to ensure it is properly closed.

Reply:
Adding, don't know if we want to keep the dump though?

Reply:
Oh yeah I forgot StringWriter is a special case that does not need a close "Closing a StringWriter has no effect. The methods in this class can be called after the stream has been closed without generating an IOException." https://docs.oracle.com/javase/8/docs/api/java/io/StringWriter.html

Reply:
I think we can get rid of the dump.  We can ask a customer to turn on jetty debug logging and get the info that way.